cycode/cli/commands/scan/code_scanner.py

```diff
@@ -476,14 +511,14 @@ def poll_scan_results(
     end_polling_time = time.time() + polling_timeout
 
     while time.time() < end_polling_time:
-        scan_details = cycode_client.get_scan_details(scan_type, scan_id)
+        scan_details = cycode_client.get_scan_details(scan_type, scan_id, should_use_scan_service)
```

Comment:
`get_scan_details` is used only in polling. polling only belongs to the scan service (AKA async scans). i think we don't need additional flags here

Reply:
We need this flag to get the correct service path

Reply:
ah ( it getting more complex and complex 

 let's not use `get_scan_service_url_path` in `get_scan_details_path` at all. to not pass every time `should_use_scan_service`. wdyt?